src/pages/settings/Report/ReportSettingsPage.js

```diff
@@ -65,7 +65,10 @@ function ReportSettingsPage(props) {
 
     const shouldDisableSettings = _.isEmpty(report) || ReportUtils.shouldDisableSettings(report) || ReportUtils.isArchivedRoom(report);
     const shouldShowRoomName = !ReportUtils.isPolicyExpenseChat(report) && !ReportUtils.isChatThread(report);
-    const notificationPreference = translate(`notificationPreferencesPage.notificationPreferences.${report.notificationPreference}`);
+    const notificationPreference =
```

Comment:
I think we should translate it and store "hidden" in notificationPreference. In case we decide to show the preference in the future. Is there any reason we want to store an empty string for notificationPreference when it is "hidden"?

Reply:
>  In case we decide to show the preference in the future.

I think when we decide to show the preference that's when we should translate it. Otherwise its just unnecessary code for this PR.

>  Is there any reason we want to store an empty string for notificationPreference when it is "hidden"?

Not sure i follow this question. We store empty string for const notificationPreference because there is nothing to show in the UI. If we attempt to translate it without values in en/es.js it will throw error. And no reason to save translations of content we are not displaying. Few [lines below](https://github.com/Expensify/App/pull/25618/files#diff-4744795365cb2b03c96513365db3891597ba5cbd1ac48e513180d4826b79d368R85-R92) you'll see that we don't render MenuItemWithTopDescription for "hidden"

Reply:
> I think when we decide to show the preference that's when we should translate it. Otherwise its just unnecessary code for this PR.

Gotchu, yeah that makes sense. I was just thinking adding the translation would simplify the code since it would remove the need for a ternary but I agree with your point about not adding a translation when we don't need it.